src/RosettaPy/utils/tools.py

```
@@ -136,3 +137,43 @@ def squeeze(items: List[T]) -> List[T]:
+        content = infile.read()
+
+    # Check if the file contains CRLF line endings
+    if "\r\n" not in content:
+        # If no CRLF line endings are found, yield the input file and exit early
+        yield input_file
+        return
+
+    # Proceed to create a temporary file and convert if CRLF exists
+    with tmpdir_manager(base_dir) as tmpdir:
+        output_file = os.path.join(tmpdir, "converted_file.txt")
+        warnings.warn(UserWarning(f"Converting CRLF line endings to LF: {input_file} -> {output_file}"))
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)